src/oaklib/cli.py

```diff
@@ -1849,8 +1849,14 @@ def descendants(terms, predicates, display: str, output_type: str, output: TextI
 @main.command()
 @click.argument("terms", nargs=-1)
 @click.option("-o", "--output")
+@click.option(
+    "--include-all-predicates/--no-include-all-predicates",
+    default=False,
+    show_default=True,
+    help="For formats that export only IS_A by default, this will include all possible predicates",
+)
 @output_type_option
-def dump(terms, output, output_type: str):
+def dump(terms, output, output_type: str, **kwargs):
```

Comment:
@joeflack4 - note here how we pass through generically all other click options. For now this is only include_all_predicates, but if more options are added, they would be passed through

Reply:
I saw this earlier; looks good! Thanks much!